CI/master_test.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+mkdir master_gold_dir
+cd master_gold_dir
+git clone https://github.com/cloud-bulldozer/cerberus.git
```

Comment:
Maybe add a check to see if the repo exists and delete if it exists just to be safe? Thoughts?

Reply:
the repo will not exist ravi, since the master_gold_dir gets created newly and deleted after every ci run .